src/custom/components/AffiliateStatusCheck/index.tsx

```diff
@@ -97,6 +103,17 @@ export default function AffiliateStatusCheck() {
       return
     }
 
+    const hasJustTraded =
+      allRecentActivity
+        .filter(isConfirmed)
+        .filter((tx: any) => tx.Date.now() - Date.parse(tx.confirmedTime) < MINUTE_MS).length >= 1
```

Comment:
Why the time range?

If there's any fulfilled order for this account it wouldn't qualify for a new affiliate link, so as long as there is any fulfilled order it should return true IMO

Unless there is another reason for that?

Reply:
I added it since it will return true even for new Affiliate links and according to what I understood from Elena's issue, if a new affiliate link is used, banner should be displayed with the text:` Invalid affiliate code: The currently connected wallet has traded before or is already part of the affiliate program.` Or am I missing something?

Reply:
No, you understood correctly. We should show this message again when a user follows another affiliate link. 
But why should we use the timer for affiliate statuses check after a successful trade? 
Within a minute or so after a successful trade I tried to paste another affiliate link and was not able to get the `Invalid affiliate code: The currently connected wallet has traded before or is already part of the affiliate program.` affiliate message. I thought it was a glitch..

Reply:
To be honest, I'm not sure how to difference between a first trade and a new affiliate link. Given that the order list is the same per user, pasting a new affiliate link within a minute works as if the user has just performed an order. Do you have any other alternative? @alfetopito @ramirotw @henrypalacios

Reply:
> But why should we use the timer for affiliate statuses check after a successful trade?
I don't know, I am asking you :)
> Within a minute or so after a successful trade I tried to paste another affiliate link and was not able to get the Invalid affiliate message. I thought it was a glitch..
Sounds like it.

Besides, the check added by Ramiro calls the /trades endpoint. I would expect that to take precedence or be used here

> To be honest, I'm not sure how to difference between a first trade and a new affiliate link.
I don't think you have to differentiate. The message cover both cases, so if the user has a trade for this account already, no new affiliate code can be applied.

Reply:
Ok perfect! So I'll remove the time filter.

Reply:
> > But why should we use the timer for affiliate statuses check after a successful trade?
>  I don't know, I am asking you :)

it was more like a rhetorical question because I do not know why and did not expect to see it in the issue fix 😌